#Lateral distance off of horizontal line, positive is left negative is right
float32 lateralDistanceFromLine
#Distance to end of line, -1 if can't see end
float32 distanceFromEnd
#Deviation angle from parallel
float32 angleToParallel

